stubs/psutil/psutil/__init__.pyi

```diff
@@ -175,7 +175,7 @@ class Process:
     info: dict[str, Any]
     def oneshot(self) -> AbstractContextManager[None]: ...
     def as_dict(
-        self, attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None, ad_value=None
+        self, attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None, ad_value: object = None
```

Comment:
```suggestion
        self,
        attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None,
        ad_value: Any = None,  # arbitrary value used as result in case of error
```